event.html

```diff
@@ -105,72 +113,72 @@ <h3 data-reactid=".832vlajev4.1.1.1.$2.$0.1">Påskemoro med barna</h3>
-    </script><script src="/javascript/app.js"></script>
+    <script data-reactid=".832vlajev4.1.3.8" type="text/javascript">
+        var _kmq = _kmq || [];
+    </script>
+</div>
+<script>
+    window.__reactAsyncStatePacket={".832vlajev4__0":{"event":{"title":"P\u00e5ske","slug":"paaske","_id":"55407a88f890929d15c3b346","__v":0,"sparks":[{"type":"spark-main","_id":"55407a88f890929d15c3b34b","items":[{"title":"P\u00e5ske = lam p\u00e5 menyen","_id":"55407a88f890929d15c3b34d","source":"matprat.no","link":"http://www.matprat.no/artikler/artikler/paske--lam-pa-menyen/","summary":"Lammekj\u00f8tt er en selvf\u00f8lge i p\u00e5sken. I p\u00e5skem\u00e5neden kj\u00f8per vi nordmenn faktisk nesten tre ganger s\u00e5 mye lammestek som i en normalm\u00e5ned. Men vet du egentlig hvor skikken med p\u00e5skelam stammer fra?","image_provider":"cloudinary","image":"5b0a9a39498108e6971e90fc471fc076"},{"title":"P\u00e5skelam - i ovn og gryte","_id":"55407a88f890929d15c3b34c","source":"matoppskrift.no","link":"http://www.matoppskrift.no/artikkel/Paaskelam-i-ovn-og-gryte","summary":"Det m\u00f8reste og saftigste lammekj\u00f8ttet f\u00e5r du ved \u00e5 langtidssteke, eller \u00e5 koke kj\u00f8ttet. ","image_provider":"cloudinary","image":"6e2407f2253160c13a47c833bac700eb"}]},{"type":"spark-default","_id":"55407a88f890929d15c3b349","items":[{"title":"Stor test: Velg riktig solkrem til barna","_id":"55407a88f890929d15c3b34a","source":"abcnyheter.no","link":"http://www.abcnyheter.no/livet/2013/03/18/stor-test-velg-riktig-solkrem-til-barna","summary":"V\u00e5r test av solkrem til barn avsl\u00f8rer at fem av dem inneholder tvilsomme ingredienser. ","image_provider":"cloudinary","image":"6defb8cd2948e0a8b57cf30b10b44ede"}]},{"type":"spark-main","_id":"55407a88f890929d15c3b347","items":[{"title":"P\u00e5skemoro med barna","_id":"55407a88f890929d15c3b348","source":"familieverden.no","link":"http://www.familieverden.no/Fritid/Forskjellig/Paskemoro-med-barna/","summary":"Barna elsker \u00e5 lage egne ting, s\u00e5 hvorfor ikke sette av en dag til kreativ utfoldelse? Her f\u00e5r du tips som garantert vil sl\u00e5 an hos de sm\u00e5 - og som er enkle \u00e5 f\u00e5 til.","image_provider":"cloudinary","image":"dd7a665393d24747add4603e37ba4422"}]}],"updatedAt":"2015-04-29T06:30:32.012Z","createdAt":"2015-04-29T06:30:32.012Z","summary":"<p>Enten du g&aring;r for byp&aring;ske, hyttep&aring;ske, skip&aring;ske eller sydenp&aring;ske, samler SOL det du trenger. Her finner du masse deilig p&aring;skestoff!<\/p>","image_provider":"cloudinary","image":"xlcxpbyl61co6jiyrpig","category":"news"}}}
+</script><script src="/javascript/app.js"></script>
 </body>
 </html>
```

Comment:
I don't know why my editor changed formatting. 
But I've changed the html markup of the default spark here.